src/routes/home/components/ProfessionDetails.tsx

```diff
@@ -35,10 +35,11 @@ const ProfessionDetails = ({ discipline }: PropsType): ReactElement => {
   const { data: progress, refresh: refreshProgress } = useReadProgress(discipline)
 
   const moduleAlreadyStarted = progress !== null && progress !== 0
-  const [documents, setDocuments] = useState<Document[] | null>(null)
-  const [trainingSet, setTrainingSet] = useState<Discipline | null>(null)
+  const [documents, setDocuments] = React.useState<Document[] | null>(null)
```

Comment:
Just wondering: Any reason why you changed this? Normally we use the other syntax and import `useState`